### PR TITLE
Added product update for 2nd September

### PR DESCRIPTION
**Pull request by sujaya-sys**

### CODE REVIEW

**Review by MartaD — APPROVED** [review]
Looks good to me

File: docs/change-log.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update September 2nd, 2022
+
+- **Improved:** The UI for [container overrides](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-container-configuration#change-container-configuration) on the [Workload Details Page](https://docs.humanitec.com/reference/user-interface/workload-details-screen#container-panel) has been reworked. Command and argument overrides are now configured via a code editor.
```
Comment (chrishumanitec) [inline docs/change-log.md:7]: > are now configured via a code editor.  

Maybe say that they can be defined directly as YAML?  Or do you think the code editor is more important?

File: docs/roadmap.md [hunk 2]
```diff
@@ -66,20 +66,6 @@ Following several months of user testing and customer feedback we are planning o
 
 Resource Drivers are used to provision things that are consumed by workloads. Most commonly this is infrastructure such as databases or dns. There are a growing number of drivers available out-of-the-box in Humanitec. In this quarter we aim to improve the flexibility and range of our driver offering. This includes adding generic drivers that work with templates and IaC as well as open-sourcing many of our drivers. We also plan to extend how drivers can be defined - allowing drivers to be defined in terms of other drivers. This should make it easier for ops teams to define good defaults to reuse within their teams. A comprehensive overview of the Humanitec driver ecosystem as well as its integration and contribution options will be published in a central hub on our website.
 
-### Q1 2022
```
Comment (chrishumanitec) [inline docs/roadmap.md:69]: Did you mean to delete this?

**Review by chrishumanitec — COMMENTED** [review]
Looks good, 2 points

**Review by chrishumanitec — APPROVED** [review]
LGTM

**Pull request merged**